akka-remote/src/test/scala/akka/remote/serialization/RandomReplacementSpec.scala

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.serialization
+
+import java.util.concurrent.Executors
+
+import akka.remote.security.provider.AESCounterBuiltinRNG
+import akka.testkit.AkkaSpec
+import org.uncommons.maths.random.{ AESCounterRNG, SecureRandomSeedGenerator }
+
+import scala.concurrent.ExecutionContext
+
+class RandomReplacementSpec extends AkkaSpec {
```

Comment:
Tests removed. I'd rather submit a separate PR for uncommons.math dependency removal